modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/APMAgentSettings.java

```diff
@@ -120,7 +118,8 @@ public void setAgentSetting(String key, String value) {
 
         // Core:
         // forbid 'enabled', must remain enabled to dynamically enable tracing / metrics
-        // forbid 'recording' / 'instrument', controlled by 'telemetry.metrics.enabled' / 'telemetry.tracing.enabled'
+        // forbid 'recording', controlled by 'telemetry.metrics.enabled' / 'telemetry.tracing.enabled'
+        "instrument",
```

Comment:
Whitelisting `instrument` so it can be enabled to investigate ES-7806

Reply:
do we need to have this change for resolving a bug? If you need to investigate this, you can make a change locally, right?

Reply:
You certainly could do that locally as well 👍 
Question is, if we want to forbid enabling automatic instrumentation. What's your preference?

Reply:
Actually, thinking about it again, it's probably better to not allow it as long as it's not working properly